plugins/azure_blob_storage/plugin.spec.yaml

```diff
@@ -412,7 +423,6 @@ actions:
         type: integer
         required: false
         example: 14
-        default: 30
```

Comment:
Any reason we're removing the default here?

Reply:
Default overwrites the example field